test/integration/backward_compatible/sql/JetTest.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable mocha/no-skipped-tests */
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const RC = require('../../RC');
+const TestUtil = require('../../../TestUtil');
+const { Client } = require('../../../../');
+
+const chai = require('chai');
+
+chai.should();
+
+describe('Jet Test', function () {
+    let client;
+    let cluster;
+    let map;
+    let map2;
+
+    const mapName = 'a';
+    const mapName2 = 'b';
+    const jetEnabledConfig = `<?xml version="1.0" encoding="UTF-8"?>
+        <hazelcast xmlns="http://www.hazelcast.com/schema/config"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.hazelcast.com/schema/config
+            http://www.hazelcast.com/schema/config/hazelcast-config-5.0.xsd">
+            <jet enabled="true"></jet>
+        </hazelcast>`;
+
+    before(async function () {
+        TestUtil.markClientVersionAtLeast(this, '5.0');
+        TestUtil.markServerVersionAtLeast(this, client, '5.0');
+    });
+
+    beforeEach(async function () {
+        map = undefined;
+        map2 = undefined;
+        cluster = await RC.createCluster(null, jetEnabledConfig);
+        await RC.startMember(cluster.id);
+        client = await Client.newHazelcastClient({
+            clusterName: cluster.id
+        });
+    });
+
+    afterEach(async function () {
+        if (map) {
+            await map.destroy();
+        }
+        if (map2) {
+            await map2.destroy();
+        }
```

Comment:
Minor: Since you destroy the cluster on each test, destroying maps are unnecessary. This way, you could also not define them above, and only define them in tests where you use maps